clients/spring-boot-starter-camunda-sdk/pom.xml

```diff
@@ -132,16 +132,6 @@
       <scope>provided</scope>
     </dependency>
 
-    <dependency>
-      <groupId>org.springframework</groupId>
-      <artifactId>spring-jcl</artifactId>
-    </dependency>
```

Comment:
🔧 This is still needed since we use `DeferredLog`, I think.

Reply:
Nope, I could savely remove it.

In fact, jcl is a "clone" of the apache commons logging.

Reply:
Ok you were right. I wonder why this did not fail locally..

Reply:
Spring JCL might come as a transitive dependency already, so you can locally use the classes without the explicit dependency. However, if the `DeferredLogger` class is from that lib, our tooling notices this and complains that you should add it explicitly.

But the [latest failure](https://github.com/camunda/camunda/actions/runs/14240819854/job/39910128152) I actually don't understand either:

You added the JCL lib and the tool complained that it's not used, right? It somehow detected that you're using a `commons-logging` class directly and didn't add this dependency? 🤔

Reply:
Spring JCL generally is super weird, I have to admit, looking into some of the issues related to it in the Spring repository 😅